include/RAJA/policy/msg_queue/spsc_queue.hpp

```diff
@@ -0,0 +1,109 @@
+/*!
+ ******************************************************************************
+ *
+ * \file
+ *
+ * \brief   Header file containing implementation for a MPSC
```

Comment:
```suggestion
 * \brief   Header file containing implementation for a SPSC
```
Would it be worth defining what these acronyms (MPSC/SPSC) mean somewhere?